cmd/release.go

```diff
@@ -139,6 +141,7 @@ func setupReleaseContext(ctx *context.Context, options releaseOpts) *context.Con
 	ctx.SkipAnnounce = ctx.Snapshot || options.skipPublish || options.skipAnnounce
 	ctx.SkipValidate = ctx.Snapshot || options.skipValidate
 	ctx.SkipSign = options.skipSign
+	ctx.SkipSBOMCataloging = options.skipSBOMCataloging
```

Comment:
should `--skip-validate` infers `--skip-sbom`?

Reply:
My take is that skipping validation shouldn't imply skipping SBOM generation. That is, cataloging artifacts isn't a form of validation and indeed creates artifacts of their own.

Reply:
hmm, yeah, that makes sense...